### ISSUE TITLE
Collapsible Headings breaks the shift-tab tooltip in Jupyter Notebook 5.0.0

### ISSUE THREAD

**Issue by jbwhit** [ipython-contrib/jupyter_contrib_nbextensions#949]
As detailed in this issue: https://github.com/jupyter/notebook/issues/2383 

Here is what it looks like on my setup: 

![example](http://g.recordit.co/KyUQ5iuJdv.gif)

And here's the output from `jupyter nbextension list`: 

```
$ jupyter nbextension list
Known nbextensions:
  config dir: /Users/jonathan/.jupyter/nbconfig
    notebook section
      collapsible_headings/main  enabled
      - Validating: OK
      codefolding/main  enabled
      - Validating: OK
      toc2/main disabled
      nbextensions_configurator/config_menu/main  enabled
      - Validating: problems found:
        - require?  X nbextensions_configurator/config_menu/main
  config dir: /Users/jonathan/miniconda3/envs/py3/etc/jupyter/nbconfig
    notebook section
      jupyter-js-widgets/extension  enabled
      - Validating: OK
    tree section
      nb_conda/tree  enabled
      - Validating: OK
  config dir: /usr/local/etc/jupyter/nbconfig
    tree section
      ipyparallel/main  enabled
      - Validating: OK
```

Other comments are stating that Collapsible Headings is where the break is coming from.

**Comment by jcb91** [comment]
Thanks for the note. So this is presumably related to a patch applied to `Tooltip._show` by collapsible_headings in order to deal with making cells `position:relative`. My guess is that the tooltip mechanism changes in 5.0.0, but the patched version is still working in the old way. I'll try to look at this in the coming few days...

**Comment by jcb91** [comment]
Presumably related to https://github.com/jupyter/notebook/commit/59458731d8ceada84832e8ed607f45e5e1bd4613

**Comment by jcb91** [comment]
yep, it seems that in 5.0.0 the patch is not needed, since the cell position is already `relative` rather than `static` as it used to be in 4.x. I've added PR #950, which should hopefully fix this. If you get a chance to try it & could let me know how it works for you, that'd be great :smile:

**Comment by wikiped** [comment]
I think `return;` [here](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/pull/950/files#diff-7667ab369d020a293efc191a40358ce1R545) breaks the Promises chain processing and the whole extension doesn't work with Jupyter 5.x. 

I tried `return Promise.resolve();` and this seems to solve the issue and keeps extension functioning as expected.

**Comment by jcb91** [comment]
d'oh, good point, I was in a rush to finish this this morning and didn't test this properly...

**Comment by jcb91** [comment]
ok, updated with the promise fix, so given you say that works for you, and it seems to work for me on 4.x, I'll go ahead and merge it soon, I think

**Comment by jcb91** [comment]
fixed in #950
